packages/test/test-pairwise-generator/src/index.ts

```diff
@@ -81,8 +99,7 @@ export function generatePairwiseOptions<T extends Record<string, any>>(optionsMa
         if(partial.__paritalKeyCount !== matrixKeys.length) {
             for(const key of matrixKeys) {
                 if(!(key in partial)) {
```

Comment:
This logic seems to assume that partials keys are a subset of the matrix keys. Any concerns here? Worth asserting that they have the same number of keys at the end?

Reply:
i'm not concerned. tests will catch it if there is a problem. rather not pay the cost of the assert